### PR TITLE
Add Coinbase Wallet Subscribe Docs

### PR DESCRIPTION
**Pull request by themacexpert**
### Description

Adds documentation for Coinbase Subscribe feature, which allows end-users to subscribe to timely updates from the Moonbeam Foundation. These updates are push notifications that get sent to the user's Coinbase Wallet app on their phone. They need to also have the Coinbase wallet extension installed on their computer to initially subscribe through the Moonbeam dApp.

### Checklist

- [x] I have added a label to this PR 🏷️
- [x] I have run my changes through Grammarly
- [x] If this requires translations for the `moonbeam-docs-cn` repo, I have created a ticket for the translations in Jira
- [ ] If pages have been moved around, I have created an additional PR in `moonbeam-mkdocs` to update redirects
- [ ] If pages have been moved around, I have run the `move-pages.py` script to move the pages and update the image paths on the chinese repo
    - [ ] After the script has been run, I have created an additional PR in `moonbeam-docs-cn`
- [ ] If images have been added, I have run the `compress-images.py` script to compress the images.
- [ ] If variables (in variables.yml) need to be updated (such as a name change), I have updated the `moonbeam-docs-cn` repo to use the new variables
- [x] If this page requires a disclaimer, I have added one

### Corresponding PRs

Please link to any corresponding PRs here.

### After Translation Requirements

- [ ] Will need to create PR in `moonbeam-docs` repo to remove images
- [ ] Will need to create PR in `moonbeam-docs` repo to remove variables
- [ ] Will need to create PR in `moonbeam-mkdocs` repo to add redirects for Chinese site
- [ ] No additional PRs are required after the translations are done

#### Items to be Updated

Please list any of the items that will need to be added or deleted after the translations are done here.

### CODE REVIEW

File: tokens/connect/coinbase-wallet.md [hunk 1]
```diff
@@ -126,6 +126,67 @@ From the **Transactions** tab, you'll be able to see your outgoing transactions,
...
+2. Press **Subscribe** in the pop-up
+
+![Subscribe in the Moonbeam dApp](/images/tokens/connect/coinbase-wallet/coinbase-12.webp)
+
+1. Press **Connect your wallet** in the resulting Coinbase pop up
+2. Next, you'll have two signature requests in your Coinbase Wallet. The first one asks you to enable XTMP Identity. Press **Sign**
+3. The next signature request is to enable the subscription to messages from the Moonbeam Foundation. Press **Sign**
+
+!!! note
+    You may receive three signature requests in the prior step if you didn't enable XMTP on your phone. The third signature request will take of that, however, you may still need to enable notifications for the Coinbase Wallet app on your phone to ensure you receive them.
```
Comment (mmaurello) [inline tokens/connect/coinbase-wallet.md:182]:
```suggestion
    You may receive three signature requests in the prior step if you didn't enable XMTP on your phone. The third signature request will take care of that, however, you may still need to enable notifications for the Coinbase Wallet app on your phone to ensure you receive them. 
```
word omission?

**Review by albertov19 — DISMISSED** [review]
LGTM - I don't have any major feedback. Thanks for the quick turnaround

File: tokens/connect/coinbase-wallet.md [hunk 2]
```diff
@@ -126,6 +126,67 @@ From the **Transactions** tab, you'll be able to see your outgoing transactions,
...
+
+1. Click the messaging icon in the upper right corner
+2. Press **Start Messaging**
+
+![Sign up for XMTP.](/images/tokens/connect/coinbase-wallet/coinbase-8.webp)
+
+Next, you'll be prompted to configure your notification settings to allow push notifications. Take the following steps:
+
+1. Press **Enable Push Notifications**
+2. Press **Allow** in the resulting iPhone prompt
```
[the remark this answers is not in the document]
Comment (themacexpert) [reply tokens/connect/coinbase-wallet.md]: fixed!

**Review by albertov19 — APPROVED** [review]
LGTM

**Pull request merged**